templates/slides.tmpl

```diff
@@ -111,15 +111,15 @@
         var textContent = el.innerText
         var match = reLanguageTag.exec(textContent)
         var languageTag
-     
+
         if (!match) {
           hljs.highlightBlock(el)
 
           return
         }
 
         languageTag = match[1]
-        el.innerText = match[2].trim()
```

Comment:
Am curious what the fail condition is on this

The parsing appeared to be allowing extra whitespace in some scenarios


But ultimately: LGTM...  suspiciously